docs/design2.md

```diff
@@ -0,0 +1,102 @@
+# Key Transparency 2.0
+
+## Objective
+
+Key Transparency currently sufferes from a design-space tradeoff in two dimentions between
+speed and verification bandwidth.  Each snapshot of the key-value merkle tree creates an additional
```

Comment:
Can you clarify which speed you mean here? (integration latency?)

Reply:
"low latency key updates"